src/Resources/config/content.xml

```diff
@@ -28,7 +28,7 @@
             </call>
 
             <call method="setCkEditorConfig">
-                <argument>%cmf_sonata_phpcr_admin_integration.content.ivory_ckeditor%</argument>
+                <argument>%cmf_sonata_phpcr_admin_integration.ivory_ckeditor.config%</argument>
```

Comment:
the name used `content` on purpose here to namespace it from other ckeditor configs